api/models/orderDao.js

```diff
@@ -0,0 +1,62 @@
+const { appDataSource } = require("../database/database");
+
+
+async function createOrderTable(userId, totalCost) {
+    try {
+			const orderTable = await appDataSource.query(
+				`
+					INSERT INTO orders 
+						(user_id, total_cost) 
+					VALUES (?, ?)
+    		`, [ userId, totalCost ]);
+
+    return orderTable;
+        
+    } catch(err) {
+        console.log("ORDER DAO1")
+        throw err;
+    } 
+}
+
+async function createOrdersRequest(orderId, products) {
```

Comment:
```suggestion
async function createOrderProducts(orderId, products) {
```
query문을 보면 order products를 create한다가 맞을 것 같네요.